openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1065,70 +1096,91 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
              "UniversityDegreeCredential"
            ]
          },
-         "proof":{
-            "proof_type":"jwt",
-            "jwt":"eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA"
-         }
+         "proofs": [
+            {
+               "proof_type":"jwt",
+               "jwt":"eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA"
+            },
+            {
+               "proof_type":"jwt",
+               "jwt":"eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
+            }
+         ]
```

Comment:
```suggestion
         "proofs": {
            "proof_type":"jwt",
            "jwt": [
               "eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA",
               "eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
            [
         }
```
What about this?

Reply:
or... 
```suggestion
         "proofs": {
            "type":"jwt",
            "values": [
               "eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA",
               "eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
            [
         }
```

but regardless this https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#name-ldp_vp-proof-type unfortunately needs to be considered or basically the fact that proof is an object for extensibility https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#section-7.2-2.2.1